backend/app/main.py

```diff
@@ -1,23 +1,35 @@
 """Routing for backend API."""
+import logging
+import os
+from logging.config import dictConfig
+
 from backend.app.requests import adequacy, providers, representative_points, service_areas
+from backend.config import config
 from backend.lib.database.postgres import connect
 from backend.lib.timer import timed
 
 import flask
 
 from flask_cors import CORS
 
+from raven.contrib.flask import Sentry
+
+
+dictConfig(config.get('logging'))
 app = flask.Flask(__name__)
 engine = connect.create_db_engine()
 
 CORS(app, resources={r'/api/*': {'origins': '*'}})
 
+sentry = Sentry(app, dsn=os.environ.get('SENTRY_DSN', None))
```

Comment:
@ericboucher not sure if you can think of a better place to put this

Reply:
I think it's fine here :)